### PR TITLE
Stop validating env var names

### PR DESCRIPTION
**Pull request by boyan-velinov**
Jira: LMCROSSITXSADEPLOY-991

### CODE REVIEW

File: com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/NameUtilTest.java [hunk 1]
```diff
@@ -18,9 +18,10 @@
 
 public class NameUtilTest {
 
-    private static final String LONG_ENV_NAME = repeat("f", NameRequirements.ENVIRONMENT_NAME_MAX_LENGTH + 1);
-    private static final List<String> VALID_ENV_NAMES = Arrays.asList("_foo", "Foo", "foo_bar", "Foo_Bar", "x_1");
-    private static final List<String> INVALID_ENV_NAMES = Arrays.asList("1foo", "   ", "foo bar", "foo-bar", "x&y", LONG_ENV_NAME);
+//    private static final String LONG_ENV_NAME = repeat("f", NameRequirements.ENVIRONMENT_NAME_MAX_LENGTH + 1);
```
Comment (KristianShishoev) [inline com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/NameUtilTest.java]: Remove comment.

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java [hunk 2]
```diff
@@ -17,11 +17,11 @@
 
     public static class NameRequirements {
 
-        public static final String ENVIRONMENT_NAME_PATTERN = "[_a-zA-Z][_a-zA-Z0-9]{0,63}";
+//        public static final String ENVIRONMENT_NAME_PATTERN = "[_a-zA-Z][_a-zA-Z0-9]{0,63}";
         public static final String XS_APP_NAME_PATTERN = "(?!sap_system)[a-zA-Z0-9\\._\\-\\\\/]{1,240}";
         public static final String CONTAINER_NAME_PATTERN = "[A-Z0-9][_A-Z0-9]{0,63}";
 
-        public static final int ENVIRONMENT_NAME_MAX_LENGTH = 64;
+//        public static final int ENVIRONMENT_NAME_MAX_LENGTH = 64;
```
Comment (KristianShishoev) [inline com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java]: Remove comment.

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java [hunk 3]
```diff
@@ -17,11 +17,11 @@
 
     public static class NameRequirements {
 
-        public static final String ENVIRONMENT_NAME_PATTERN = "[_a-zA-Z][_a-zA-Z0-9]{0,63}";
+//        public static final String ENVIRONMENT_NAME_PATTERN = "[_a-zA-Z][_a-zA-Z0-9]{0,63}";
```
Comment (KristianShishoev) [inline com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java]: Remove comment.

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java [hunk 4]
```diff
@@ -46,11 +46,10 @@ public static boolean isValidName(String name, String namePattern) {
         return name.matches(namePattern);
     }
 
-    public static void ensureValidEnvName(String name, boolean shouldAllowInvalidEnvNames) {
-        if (name.matches(NameRequirements.ENVIRONMENT_NAME_PATTERN) || shouldAllowInvalidEnvNames) {
-            return;
+    public static void ensureValidEnvName(String name) {
```
Comment (KristianShishoev) [inline com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java]: Move this block into invocation method.

Comment (boyan-velinov) [reply com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java]: Done

File: com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/NameUtilTest.java (hunk 1 above)
Comment (boyan-velinov) [reply com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/NameUtilTest.java]: Done

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java (hunk 2 above)
Comment (boyan-velinov) [reply com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java]: Done

File: com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java (hunk 3 above)
Comment (boyan-velinov) [reply com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/NameUtil.java]: Done

**Pull request merged**